inat_to_cams/cams_writer.py

```diff
@@ -136,10 +138,18 @@ def write_feature(self, cams_feature, inat_id, existing_feature, dry_run):
             ('Species', cams_feature.weed_location.species),
             ('DataSource', cams_feature.weed_location.data_source),
             ('Location details', cams_feature.weed_location.location_details),
+            ('Reported Longitude', cams_feature.weed_location.reported_longitude),
+            ('Reported Latitude', cams_feature.weed_location.reported_latitude),
             ('Effort to control', cams_feature.weed_location.effort_to_control),
             ('CurrentStatus', cams_feature.weed_location.current_status),
             ('iNaturalistURL', cams_feature.weed_location.external_url)
         ]
+        if not weed_geolocation_modified:
+            fields.remove(('Reported Longitude', cams_feature.weed_location.reported_longitude))
+            fields.remove (('Reported Latitude', cams_feature.weed_location.reported_latitude))
+            new_layer_row[0].pop('geometry')
```

Comment:
Ok. did this, but do you want to take a look and see if it could be smarter. My python is a bit lacking when it comes to some of these strucutre manipulations.